data/remnant/remnant.txt

```diff
@@ -276,6 +276,490 @@ phrase "remnant"
 		"Soaring"
 		"Enduring"
 
+phrase "remnant uncontacted"
+	word
+		"(They sent you a hail consisting of a sign language you don't understand.)"
+
+phrase "remnant uncontacted"
+	word
+		"I am"
+		"We are"
+	word
+		" "
+	word
+		"watching"
+		"observing"
+		"analyzing"
+		"overseeing"
+	word
+		" "
+	word
+		"you"
+		"your actions"
+		"your behavior"
+	word
+		"."
+
+phrase "remnant uncontacted hostile"
+	word
+		"Eat space dust"
+		"Time to go"
+		"Sing your last prayers"
+		"Leave this space"
+	word
+		","
+	word
+		" "
+	word
+		"hostile ship"
+		"pirate"
+		"Alpha"
+	word
+		"!"
+
+phrase "remnant uncontacted hostile"
+	word
+		"I"
+		"My friend"
+		"My comrade"
+		"My gut"
+	word
+		" "
+	word
+		"was right"
+		"was correct"
+		"knew it"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"You're"
+	word
+		" "
+	word
+		"hostile"
+		"here with malicious intent"
+		"an Alpha"
+	word
+		"!"
+
+phrase "remnant uncontacted hostile"
+	word
+		"(They sent you hail consisting of a sign language you don't understand, and it doesn't seem friendly.)"
+		"(They sign so quickly that your hail receiver can't process it all.)"
+		"(It seems you are not welcome here.)"
+
+phrase "friendly remnant"
+	word
+		"Greetings"
+		"Howdy"
+		"Hello there"
+		"Salutations"
+	word
+		", "
+		" "
+	word
+		"human"
```

Comment:
While the Remnant do free the Alphas, I don't think that that would then lead them to greeting each other as "humans."
```suggestion
```